packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from '@common/intl';
+import {
+  BasicTextInput,
+  CircularProgress,
+  Grid,
+  LoadingButton,
+  NumericTextInput,
+  SaveIcon,
+  Typography,
+  UpdateSyncSettingsInput,
+  useNotification,
+} from '@openmsupply-client/common';
+import { useHost } from '../api/hooks';
+import { Setting } from './Setting';
+
+type SyncSettings = Omit<UpdateSyncSettingsInput, '__typename'>;
+
+const isValid = (syncSettings: SyncSettings | null) => {
+  if (!syncSettings) return false;
+  return (
+    !!syncSettings.url &&
+    !!syncSettings.username &&
+    !!syncSettings.password &&
+    !!syncSettings.intervalSec &&
+    !!syncSettings.centralServerSiteId &&
+    !!syncSettings.siteId
+  );
+};
+
+const SyncSettingsForm = ({
+  isDisabled,
+  isSaving,
+  isValid,
+  settings,
+  onSave,
+  setSyncSettings,
+}: {
+  isDisabled: boolean;
+  isSaving: boolean;
+  isValid: boolean;
+  settings?: SyncSettings;
+  onSave: () => void;
+  setSyncSettings: (syncSettings: SyncSettings) => void;
+}) => {
+  const t = useTranslation('common');
+  interface SyncSettingProps {
+    autocomplete?: string;
+    property: keyof SyncSettings;
+    type?: string;
+  }
+
+  const StringSyncSetting: FC<SyncSettingProps> = ({
+    property,
+    type = 'text',
+    autocomplete = 'off',
+  }) => {
+    const value = settings?.[property] || '';
+    const onChange = (value: string) => {
+      const patched = { ...settings, [property]: value } as SyncSettings;
+      setSyncSettings(patched);
+    };
+    return (
+      <BasicTextInput
+        value={value}
+        onChange={e => onChange(e.target.value)}
+        type={type}
+        inputProps={{ autoComplete: autocomplete }}
+        disabled={isDisabled}
+      />
+    );
+  };
+
+  const NumericSyncSetting: FC<SyncSettingProps> = ({ property }) => {
+    const value = settings?.[property] || '';
+    const onChange = (value: string) => {
+      const patched = {
+        ...settings,
+        [property]: Number(value),
+      } as SyncSettings;
+      setSyncSettings(patched);
+    };
+    return (
+      <NumericTextInput
+        value={value}
+        onChange={e => onChange(e.target.value)}
+        disabled={isDisabled}
+      />
+    );
+  };
+
+  return (
+    <form style={{ width: '100%' }}>
+      <Setting
+        title={t('label.settings-url')}
+        component={<StringSyncSetting property="url" />}
+      />
+      <Setting
+        title={t('label.settings-username')}
+        component={<StringSyncSetting property="username" />}
+      />
+      <Setting
+        title={t('label.settings-password')}
+        component={
+          <StringSyncSetting
+            property="password"
+            type="password"
```

Comment:
type="password" to obscure the input chars, and autocomplete to keep chrome happy ( raises a console warning otherwise )